.github/workflows/golang-ci-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+name: Golang CI Workflow
+
+on: push
+
+jobs:
+  ci:
+    name: golang-ci
+    runs-on: ubuntu-latest
+    env:
+      COVERAGE_PROFILE_OUTPUT_LOCATION: "./profile.cov"
+    steps:
+      - name: Checkout code / lint code / install dependencies for goveralls / run tests
+        uses: uc-cdis/.github/.github/actions/golang-ci@master
```

Comment:
the reusable workflow runs the tests, so we can stop running them in travis - could you delete https://github.com/uc-cdis/indexs3client/blob/master/.travis.yml

Reply:
thanks for catching that!